converters/fromZigbee.js

```diff
@@ -616,6 +616,24 @@ const converters = {
             return result;
         },
     },
+    meter_identification: {
+        cluster: 'haMeterIdentification',
+        type: ['readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+            const elements = [
```

Comment:
The guideline is that returned payloads are snake_case, I see many camelCased properties here. A simple regex might do it: https://www.codegrepper.com/code-examples/javascript/js+camelcase+to+snake+case

Reply:
I don't follow you. The only camel thing there is "ReadResponse" (which I supuse is a constant from other place). The rest are Attribute and Cluster names, which come from ZCL/zm directly. I just match them for filtering. Correct me if I'm wrong

BR,
Valentin

Reply:
In the for loop below you iterate over the `elements`, if e.g. `at` is ``powerThreshold`, you will set `result.powerThreshold = msg.data['powerThreshold']`, the properties of `result` should be snake_cased, not camelCase. (so it should be `result.power_threshold` in this case)

Reply:
Thank you for clarifying it. Now it should be ok